lib/connection_pool/timed_stack.rb

```diff
@@ -118,6 +129,27 @@ def length
     @max - @created + @que.length
   end
 
+  ##
+  # Pre-create all possible connections
+  def fill
```

Comment:
What is a purpose of these methods `fill` and `add_one`? I don't see them being used.
Also in what case I would want to start whole pool? Is it for testing purpose?